egs2/mediaspeech/asr1/RESULTS.md

```diff
@@ -0,0 +1,29 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
```

Comment:
- Can you merge this to README.md?
- Upload a model to the HF hub and put a link here.

Reply:
Done.